Lib/gfsubsets/data/bengali_unique-glyphs.nam

```diff
@@ -104,10 +106,21 @@
 0x09FC BENGALI LETTER VEDIC ANUSVARA
 0x09FD BENGALI ABBREVIATION SIGN
 0x09FE BENGALI SANDHI MARK
+0x1CD0 VEDIC TONE KARSHANA
+0x1CD2 VEDIC TONE PRENKHA
+0x1CD5 VEDIC TONE YAJURVEDIC AGGRAVATED INDEPENDENT SVARITA
+0x1CD6 VEDIC TONE YAJURVEDIC INDEPENDENT SVARITA
+0x1CD8 VEDIC TONE CANDRA BELOW
+0x1CE1 VEDIC TONE ATHARVAVEDIC INDEPENDENT SVARITA
+0x1CEA VEDIC SIGN ANUSVARA BAHIRGOMUKHA
+0x1CED VEDIC SIGN TIRYAK
+0x1CF2 VEDIC SIGN ARDHAVISARGA
+0x1CF5 VEDIC SIGN JIHVAMULIYA
+0x1CF6 VEDIC SIGN UPADHMANIYA
 0x1CF7 VEDIC SIGN ATIKRAMA
-0x1CFA VEDIC SIGN DOUBLE ANUSVARA ANTARGOMUKHA
```

Comment:
I've seen a few instances in the Indic scripts where some characters are being removed, is this intended? Just want to make sure we're not breaking anything.

Reply:
As mentioned above:

> For example, when we had sample texts in transliterated Sanskrit, we tried adding things like DEVANAGARI SIGN AVAGRAHA to various subsets to try to make them work. That wasn't a good idea and they should be removed

This particular codepoint doesn't have a Bengali script extension, so it could never be used in a Bengali script-itemised run.

Reply:
Sounds good.